docker/utils/utils.py

```diff
@@ -725,6 +725,15 @@ def create_host_config(binds=None, port_bindings=None, lxc_conf=None,
 
         host_config['SecurityOpt'] = security_opt
 
+    if sysctls:
+        if not isinstance(sysctls, dict):
+            raise host_config_type_error('sysctls', sysctls, 'dict')
+        host_config['Sysctls'] = {}
+        for k, v in six.iteritems(sysctls):
+            if not isinstance(v, six.string_types):
+                v = str(v)
```

Comment:
I think we should use `six.text_type` here - `str` means different things in Python 2/3.